src/components/Login/JellyfinLogin.tsx

```diff
@@ -1,63 +1,39 @@
 import Button from '@app/components/Common/Button';
```

Comment:
I moved the Jellyfin setup to its own component (`JellyfinSetup.tsx`), since previously the entire `JellyfinLogin` component was just an if statement over the `initial` flag.